### ISSUE TITLE
Crash when starting SD Maid

### ISSUE THREAD

**Issue by d4rken** [d4rken/sdmaid-public#361]
Happens for a few select types of devices:

* Siswoo C50
* Coolpad Coolpad 8702d
* Prestigio Psp3503duo
* Prestigio Psp3509duo
* Thl Thl t11
* Intel Ocean mini 

MediaTek devices again? *sigh*

```
java.lang.RuntimeException: Unable to create service eu.thedarken.sdm.SDMService: android.os.BadParcelableException: ClassNotFoundException when unmarshalling: eu.thedarken.sdm.lib.external.ExternalEventSet
        at android.app.ActivityThread.handleCreateService(ActivityThread.java:3005)
        at android.app.ActivityThread.access$1800(ActivityThread.java:178)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1553)
        at android.os.Handler.dispatchMessage(Handler.java:111)
        at android.os.Looper.loop(Looper.java:194)
        at android.app.ActivityThread.main(ActivityThread.java:5637)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:959)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:754)
Caused by: android.os.BadParcelableException: ClassNotFoundException when unmarshalling: eu.thedarken.sdm.lib.external.ExternalEventSet
        at android.os.Parcel.readException(Parcel.java:1548)
        at android.os.Parcel.readException(Parcel.java:1499)
        at android.app.ActivityManagerProxy.broadcastIntent(ActivityManagerNative.java:2922)
        at android.app.ContextImpl.sendBroadcast(ContextImpl.java:1482)
        at android.content.ContextWrapper.sendBroadcast(ContextWrapper.java:382)
        at eu.thedarken.sdm.SDMService.postClearWidgets(SDMService.java:314)
        at eu.thedarken.sdm.SDMService.onCreate(SDMService.java:153)
        at android.app.ActivityThread.handleCreateService(ActivityThread.java:2995)
        at android.app.ActivityThread.access$1800(ActivityThread.java:178)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1553)
        at android.os.Handler.dispatchMessage(Handler.java:111)
        at android.os.Looper.loop(Looper.java:194)
        at android.app.ActivityThread.main(ActivityThread.java:5637)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:959)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:754)
```

**Comment by d4rken** [comment]
Possibly similiar https://code.google.com/p/android/issues/detail?id=6822

**Comment by d4rken** [comment]
The bytearray trick also worked here. Seems like these specific devices parcel or reparcel the bundle outof process?

**Comment by d4rken** [comment]
Fixed in v4.1.5
